frontend/app/src/form/index.tsx

```diff
@@ -81,6 +142,12 @@ export default function Form(props: any) {
       case 3:
         setSchemaData(BountyDetailsCreationData.step_3);
         break;
+      case 4:
```

Comment:
do you think we can use an array in the `BountyDetailsCreationData` so we don't need to keep changing this bit of code an instead just need to change the values?

Reply:
Sure, I'll look into this.